graphql-dgs-subscriptions-sse/src/main/kotlin/com/netflix/graphql/dgs/subscriptions/sse/DgsSSESubscriptionHandler.kt

```diff
@@ -41,7 +42,6 @@ import java.util.*
  */
 @RestController
 open class DgsSSESubscriptionHandler(open val dgsQueryExecutor: DgsQueryExecutor) {
-    open val logger = LoggerFactory.getLogger(DgsSSESubscriptionHandler::class.java)
```

Comment:
@lucatk, any chance you recall any reason to make the logger `open`?

Reply:
to allow CGLIB proxying of DGS' Bean classes, see #428.
I've not explored "companion objects" yet, so I can't verify whether or not this will break CGLIB proxying if you keep the companion object private.

Reply:
This should not affect CGLIB proxies since the logger will not be an attribute of the reference but the class. You can see this all over the place in Java where loggers are private and static.